src/main/java/com/minecolonies/commands/AbstractSplitCommand.java

```diff
@@ -14,7 +15,7 @@
 /**
  * A command that has children. Split-command with various parts.
  */
-public abstract class AbstractSplitCommand implements ISubCommand
+public abstract class AbstractSplitCommand extends AbstractSingleCommand implements ISubCommand
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) "ISubCommand" is implemented by a super class; there is no need to implement it here. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1939)